lib/pages/login_page.dart

```diff
@@ -120,6 +122,10 @@ class _LoginPageState extends ConsumerState<LoginPage> {
     );
   }
 
+  void showOnlyLoginScreen(BuildContext context) {
```

Comment:
imo bez sensu do osobnej funkcji takie pojedyńcze calle brać bo to bardziej miesza niż pomaga ale good point żeby więcej niż 1 pop

Reply:
może powinniśmy generic funkcje mieć która robi popUntil widget.routeName czy coś

Reply:
no imo wywolanie funkcji z taka nazwa jest zdecydowanie ladniejsze niz tego co jest wewnatrz - `Navigator.of(context).popUntil((ro ...`